www/source/partials/docs/_bp-kubernetes.html.md.erb

```diff
@@ -3,7 +3,15 @@
 
 ## Habitat Kubernetes Operator
 
-The [Habitat Kubernetes Operator](https://github.com/kinvolk/habitat-operator) is on-going work to create an operator that leverages Kubernetes API services to create a native and robust integration between the two technologies. Follow along on [github](https://github.com/kinvolk/habitat-operator), and join us in our #kubernetes channel in the [Habitat Slack Channel](https://slack.habitat.sh) for more information.
+The [Habitat Kubernetes Operator](https://github.com/kinvolk/habitat-operator) is on-going work to create an operator that leverages Kubernetes API services to create a native and robust integration between the two technologies. Follow along on [github](https://github.com/kinvolk/habitat-operator), and join us in our [#kubernetes](https://habitat-sh.slack.com/messages/kubernetes/) channel in the [Habitat Slack](https://slack.habitat.sh) for more information.
+
+Packages can be easily made to be run on Kubernetes using the [Kubernetes exporter]:
+
+```shell
+$ hab pkg export kubernetes ORIGIN/NAME
+```
+
+For more information on the Kubernetes exporter, check out [this blog post](https://kinvolk.io/blog/2017/12/introducing-the-habitat-kubernetes-exporter/).
 
 ## Docker and ACI
 
```

Comment:
There is a `## kubectl` a bit below here. Super confusing for the user IMO. First we say, something about habitat operator, then the `kubectl run ...` stuff. Maybe we could restructure this in some way?

Reply:
Should we just remove the `kubectl` section here? Not sure how to succintly explain that there are 3 (!) options:

* kubectl
* k8s exporter
* helm exporter

Reply:
Maybe restructure the Kubernetes section in two, one with habitat-operator and one without. Something like that would make sense to me:

# Kubernetes
## Habitat operator
// Explain why running with habitat operator is good, e.g. things are managed for you, you can update configuration, all the other advantages...
// Then all the other two exporters

## No operator ?
// You can just deploy application with the `kubectl run` that you exported from docker image...

Just a suggestion, WDYT?